### PR TITLE
feat: add wallet-automation

### LINKED ISSUE #493: 🛠️ [TASK] : Write initial integration tests using Typhon and Lace

### PR DESCRIPTION
**Pull request by SotaTek-DuyLe**
# Description

This PR introduces UI tests for the cardano_catalyst package using Typhon and Lace wallets.

Close #493

### CODE REVIEW

File: .config/dictionaries/project.dic [hunk 1]
```diff
@@ -76,6 +79,7 @@ fontello
 formz
 Formz
 fuzzer
+gafhhkghbfjjkeiendhlofajokpaflmk
```
Comment (dtscalac) [inline .config/dictionaries/project.dic:82]: You're sure this is a valid word? Probably would be better just to disable it for the lines that break the cspell, i.e.:

```
/* cSpell:disable */
final myVar = 'gafhhkghbfjjkeiendhlofajokpaflmk';
/* cSpell:enable */
```

File: catalyst_voices_packages/catalyst_cardano/catalyst_cardano/test/wallet-automation/keys.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# .txt file
+//WALLET1
+WALLET1_USERNAME=test123
+WALLET1_PASSWORD=test12345678@
+WALLET1_SEED_WORD_1=stomach
```
Comment (dtscalac) [inline catalyst_voices_packages/catalyst_cardano/catalyst_cardano/test/wallet-automation/keys.txt:5]: One consideration, the approach of having each word in a separate var means that you need to know in advance the number of words when you read it or you need to look for WALLET1_SEED_WORD_{n} pattern.

I think many programming languages allow to split a string by a comma, etc.

To allow different seed lengths I would keep all the words as the following:

```
WALLET1_SEED_WORDS=stomach,horn,rail,afraid,...
```

and later in the code:
```
let seed = readArg("WALLET1_SEED_WORDS")
let words = seed.split(",")
```

What do you think?

File: catalyst_voices_packages/catalyst_cardano/catalyst_cardano/example/lib/sign_and_submit_rbac_tx.dart [hunk 3]
```diff
@@ -109,7 +109,7 @@ Future<void> _signAndSubmitRbacTx({
 
     final txHash = await api.submitTx(transaction: signedTx);
     result = 'Tx hash: ${txHash.toHex()}';
-  } catch (error) {
+   } catch (error) {
```
Comment (dtscalac) [inline catalyst_voices_packages/catalyst_cardano/catalyst_cardano/example/lib/sign_and_submit_rbac_tx.dart:112]: Pls reformat this file, I think the indentation is corrupted

**Review by dtscalac — APPROVED** [review]
LGTM

Somebody with a stronger TS experience could have a look at TS files.

**Comment by minikin** [comment]
@kukkok3, can we please update the ticket description?

**Review by dtscalac — APPROVED** [review]

**Pull request merged**